templates/mentorship_relation_accepted.html

```diff
@@ -1,11 +1,11 @@
-<p>Hi {{ request_sender }},</p>
+<h3 style="color:#8d2165;">Hi {{ request_sender }},</h3>
 <br>
 <p>Congratulations!</p>
 <br>
-<p>Your mentorship request has been accepted by {{ request_receiver }},
-    to be your {{ role }} until {{ end_date }}.
-    Wish you a great mentorship relation with {{ request_receiver }}.</p>
+<p><Span style="color: #005c88;">Your mentorship request has been accepted by <strong>{{ request_receiver }}</strong>
+    to be your <Span style="color: #005c88;"><strong>{{ role }}</strong></Span> until <Span style="color: #005c88;"><strong> {{ end_date }}</strong></Span>.
+    <Span style="color: #005c88;">Wish you a great mentorship relation with <strong>{{ request_receiver }}</strong></Span>.</p>    
 <br>
 <p><i>In Systerhood,</i></p>
-<p>Systers Open Source</p>
-<p><a href="http://systers.io/">http://systers.io/</a></p>
+<p>Systers Open Source</p>	<p style="color:#8d2165;"><strong>Systers Open Source</strong></p>
```

Comment:
Duplicate `Systers Open Source` (remove one)

Reply:
Thanks ! Just made the changes